client/app/visualizations/word-cloud/index.js

```diff
@@ -9,7 +9,7 @@ function findWordFrequencies(data, columnName) {
   const wordsHash = {};
 
   data.forEach((row) => {
-    const wordsList = row[columnName].toString().split(' ');
+    const wordsList = typeof row[columnName] === 'string' ? row[columnName].toString().split(' ') : '';
```

Comment:
If we know it's a string, then why do we need to call `toString`?

Reply:
Whoops, my bad. Fixed it.